### PR TITLE
Add guide for isOwner policy

### PR DESCRIPTION
**Pull request by lauriejim**
#### Description of what you did:

Created a guide to restrict entry update for the author only.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: docs/3.0.0-beta.x/guides/is-owner.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        ctx.params,
+        ctx.request.body
+      );
+    }
+
+    return sanitizeEntity(entity, { model: strapi.models.article });
+  },
+};
+```
+
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/guides/is-owner.md]: You should mention or implement the restriction on delete too don't you think ?

Comment (lauriejim) [reply docs/3.0.0-beta.x/guides/is-owner.md]: I think I can add a note for the delete (same code than the update)

Comment (lauriejim) [reply docs/3.0.0-beta.x/guides/is-owner.md]: Done

File: docs/3.0.0-beta.x/guides/is-owner.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# Create is owner policy
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/guides/is-owner.md]: # Create is owner policy

This guide will explain how to restrict content edition to content authors only.

## Introduction

It is often required that the author of an entry is the only user allowed to edit or delete the entry.

This is a feature that is requested a lot and in this guide we will see how to implement it.

## Apply the author by default

When we are creating a new Article via `POST /articles` we will need to set the authenticated user as the author of the article.

To do so we will customize the `create` controller function of the Article API.

Comment (lauriejim) [reply docs/3.0.0-beta.x/guides/is-owner.md]: DOne

**Pull request merged**